.buildkite/data.rayci.yml

```diff
@@ -66,10 +66,31 @@ steps:
     depends_on: datanbuild
     job_env: forge
 
-  - label: ":database: data: doc gpu"
+  - label: ":database: data: doc tests"
     tags: 
       - data
-      - ci_doc
+      - doc
+    instance_type: medium
+    commands:
+      # doc tests
+      - bazel run //ci/ray_ci:test_in_docker -- python/ray/... //doc/... data 
+        --build-name data12build
+        --except-tags gpu
+        --only-tags doctest
+        --parallelism-per-worker 2
+      # doc examples
+      - bazel run //ci/ray_ci:test_in_docker -- //doc/... data 
+        --build-name data12build
+        --except-tags gpu,post_wheel_build,doctest
+        --parallelism-per-worker 2
+        --skip-ray-installation
```

Comment:
should these two lines be in two separate jobs?

or why are we grouping them together?

Reply:
they are grouped based on coverage; otherwise i tend to group things together in one job

unless there are compelling reason to separate them, running in one job save the bootstraping overhead (downloading docker image + building ray)